engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/views/discovery-create.html

```diff
@@ -1,91 +1,89 @@
 <span page-title>{{ 'Create Repositories' | translate }}</span>
 
-<form name="productForm" class="col-sm-5" ng-disabled="creating()" novalidate role="form">
+<form name="productForm" class="col-sm-5 form-horizontal" ng-disabled="creating()" novalidate role="form">
```

Comment:
We are moving away from using `.form-horizontal`.  Do you mind changing this to use the normal bs3 form?